README.md

```diff
@@ -70,57 +79,82 @@ Use the following commands to set up the `brieflow_configuration` Conda environm
+
+The steps for running workflows currently include local and Slurm integration.
+To use the Slurm integration for Brieflow configure the Slurm resources in [analysis/slurm/config.yaml](analysis/slurm/config.yaml).
+The `slurm_partition` and `slurm_account` in `default-resources` need to be configured while the other resource requirements have suggested values.
+These can be adjusted as necessary.
+
+**Note**: Other Snakemake HPC integrations can be found in the [Snakemake plugin catalog](https://snakemake.github.io/snakemake-plugin-catalog/index.html#snakemake-plugin-catalog).
+Only the `slurm` plugin has been tested.
+
+### Analysis Steps
 
 Follow the instructions below to configure paramers and run wokflows.
-All of these steps are done in the example analysis folder.
+All of these steps are done in the analysis folder.
 Use the following command to enter this folder:
-`cd example_analysis`.
+`cd analysis/`.
 
-#### Step 2.0: Configure preprocess params
+#### Step 0: Configure preprocess params
 
-Follow the steps in [0.configure_preprocess_params.ipynb](example_analysis/0.configure_preprocess_params.ipynb) to configure preprocess params.
+Follow the steps in [0.configure_preprocess_params.ipynb](analysis/0.configure_preprocess_params.ipynb) to configure preprocess params.
 
-#### Step 2.1: Run preprocessing workflow
+#### Step 1: Run preprocessing workflow
 
+**Local**:
 ```sh
 conda activate brieflow_workflows
 sh 1.run_preprocessing.sh
 ```
+**Slurm**:
+```sh
+sh 1.run_preprocessing_slurm.sh
+```
 
-#### Step 2.2: Configure SBS/phenotype process params
+#### Step 2: Configure SBS process params
 
-1) Follow the steps in [2.configure_sbs_process_params.ipynb](example_analysis/2.configure_sbs_process_params.ipynb) to configure SBS process params.
-2) Follow the steps in [3.configure_phenotype_process_params.ipynb](example_analysis/3.configure_phenotype_process_params.ipynb) to configure phenotype process params.
+Follow the steps in [3.configure_phenotype_process_params.ipynb](analysis/3.configure_phenotype_process_params.ipynb) to configure phenotype process params.
 
-#### Step 2.3: Run SBS/phenotype process workflow
+#### Step 3: Configure phenotype process params
 
+Follow the steps in [2.configure_sbs_process_params.ipynb](analysis/2.configure_sbs_process_params.ipynb) to configure SBS process params.
+
+#### Step 4: Run SBS/phenotype process workflow
+
+**Local**:
 ```sh
 conda activate brieflow_workflows
 sh 4.run_sbs_phenotype_processes.sh
 ```
+**Slurm**:
+```sh
+sh 4.run_sbs_phenotype_processes_slurm.sh
+```
 
 ***Note**: Use `brieflow_configuration` Conda environment for each configuration notebook.
 
-
-### Run entire example analysis
+### Run Entire Analysis
 
 If all parameter configurations are known for the entire Brieflow pipeline, it is possible to run the entire pipeline with the following:
 
 ```sh
-cd example_analysis
+cd analysis/
 sh run_entire_analysis.sh
 ```
 
+### Example Analysis
 
-## Contribution Notes
```

Comment:
The only thing I would add is some note about issues and that we're open to contributions at this early stage--and maybe some documentation that tells you how to do this?

Reply:
Done in https://github.com/cheeseman-lab/brieflow/pull/28/commits/58f22adfa2b4afacbde5654716c84a01fcd2707a